### ISSUE TITLE
fix: `&caps_word` randomly deactivates 

### ISSUE THREAD

**Issue by jjangga0214** [zmkfirmware/zmk#1708]
Hi! 

I observed `&caps_word` *randomly* deactivates. 
(Apology for not giving a well-defined, reproducible example, as it seems random.)
It suddenly deactivates from an arbitrary point in the middle of the word.
For instance, it is written like `ZMKFIRmware`.

Once it deactivates, if I try again, only the word's first letter is capitalized **for a while** (e.g. for about 5 to 10 minutes).
For instance, it is written like `Zmkfirmware`.

Is it only me experiencing this?

Thanks :)

**Comment by jjangga0214** [comment]
@caksoylar Thanks for the reference :) 
But mine is macOS Ventura 13.2.1. And I don't use Remote Desktop or VM. 🧐

**Comment by jjangga0214** [comment]
@caksoylar  I found the reasons. 

One of them is that I set my keyboard layout as qwerty from zmk point of view(keyboard level), but remapped it to Colemak from the OS level. Thus while typing, I thought I am writing alphabet 'O', but it was actually semicolon from zmk side. So everytime I press 'O', caps_word deactivated, which was the right behavior. I solved this problem by input colon(shift+semicolon) over semicolon while `&caps_word` like this way(https://github.com/zmkfirmware/zmk/issues/1706#issuecomment-1472128865). 

The other reason was related to another personal testing environment, which is not zmk's fault. 

Sorry and thanks again.
